docs/playground/ko/JavaScript/Functions with JavaScript/Function Chaining.ts

```diff
@@ -0,0 +1,81 @@
+//// { order: 2, compiler: { esModuleInterop: true } }
+
+// 함수 체이닝 API는 JavaScript의 흔한 패턴입니다.
+// 이는 여러분의 코드를 
+// 중간값을 덜 포함함으로써 집중시킬 수 있고
+// JavaScript 중첩 특성 덕분에 더 읽기 쉽게 만들 수 있습니다.
+
+// 체이닝으로 동작하는 실제로 흔한 API는 jQuery입니다.
```

Comment:
```suggestion
// 체이닝으로 동작하는 흔한 API는 jQuery입니다.
```
"실제로"가 어색해서 지워봤습니다 :)